.pipelines/templates/e2e-windows-template.yaml

```diff
@@ -53,7 +55,8 @@ jobs:
           export WINDOWS_PACKAGE_VERSION=$KUBERNETES_VERSION
         fi
         export K8S_VERSION=${WINDOWS_PACKAGE_VERSION//./}
-        export WINDOWS_E2E_STORAGE_CONTAINER=${{ parameters.storageAccount }}-$K8S_VERSION
+        declare -l WINDOWS_E2E_STORAGE_CONTAINER=${{ parameters.storageAccount }}-$K8S_VERSION
```

Comment:
Do we cleanup the storage account and containers?

Reply:
I found some code but it only touches ws2019
    if [ "$WINDOWS_E2E_IMAGE" == "2019-containerd" ]; then
        cleanupOutdatedFiles
    fi

Reply:
We don't cleanup the storage account. We just cleanup the containers when the related k8s version is not supported any more in https://github.com/Azure/AgentBaker/blob/master/e2e/windows/e2e-helper.sh#L19